runtime/near-vm-errors/src/lib.rs

```diff
@@ -363,12 +382,13 @@ impl fmt::Display for FunctionCallError {
             FunctionCallError::HostError(e) => e.fmt(f),
             FunctionCallError::LinkError { msg } => write!(f, "{}", msg),
             FunctionCallError::WasmTrap(trap) => write!(f, "WebAssembly trap: {}", trap),
-            FunctionCallError::WasmUnknownError => {
-                write!(f, "Unknown error during Wasm contract execution")
+            FunctionCallError::WasmUnknownError { debug_message } => {
+                write!(f, "Unknown error during Wasm contract execution: {}", debug_message)
```

Comment:
Still feel we should rename this to WasmInternalError, it's a paradox that it's unknown then error detail (debug_message) is given

Reply:
Any renames like that would lead to serialization issues again. Think is general we shall go in direction of separatimg runtime and serialized data structures.